src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -140,6 +142,50 @@ static constexpr std::chrono::minutes MAX_LEDGER_AGE_ACQUIRE {1};
 // Don't acquire history if write load is too high
 static constexpr int MAX_WRITE_LOAD_ACQUIRE {8192};
 
+// Helper function for LedgerMaster::doAdvance()
+// Returns the minimum ledger sequence in SQL database, if any.
+boost::optional<LedgerIndex>
```

Comment:
```
static
boost::optional<LedgerIndex>
```

Reply:
Same, this is a free function, not a class member.